packages/url-loader/src/constants/parameters.ts

```diff
@@ -220,4 +219,14 @@ export const zoom = {
       text: 'Controls how close to crop to the detected coordinates when using face-detection, custom-coordinate, or object-specific gravity.',
       url: 'https://cloudinary.com/documentation/transformation_reference#z_zoom'
     })),
-}
\ No newline at end of file
+}
+
+
+/** Base Resizing */
+
+export const baseAspectRatio = { ...aspectRatio };
+export const baseCrop = { ...crop };
+export const baseGravity = { ...gravity };
+export const baseHeight = { ...height };
+export const baseWidth = { ...width };
+export const baseZoom = { ...zoom };
```

Comment:
question: could you explain what is the reason for these base variables? AFAIK they just export the regular resizing options like aspectRatio, crop, etc?

Can't we just use the `aspectRadio, etc` instead of these base variables?

Reply:
updated this. they use a shared schema definition now but use distinct descriptions

Reply:
Ok, thanks :)